requirements.in

```diff
@@ -3,6 +3,7 @@ boto3
 google-cloud-storage
 cachetools
 click
+dataclasses
```

Comment:
I am not sure you need to add `dataclasses` here: unless I am mistaken, it is part of stdlib since 3.7.

Reply:
you are right

Reply:
PTAL